packages/model-viewer/src/model-viewer-base.ts

```diff
@@ -308,6 +326,12 @@ export default class ModelViewerElementBase extends UpdatingElement {
       // at all times:
       this[$isElementInViewport] = true;
     }
+    const ModelViewerGlobalConfig: ModelViewerGlobalConfig =
+        (self as any).ModelViewerElement || {};
+    if (ModelViewerGlobalConfig.powerPreference != null) {
+      ModelViewerElementBase.powerPreference =
+          ModelViewerGlobalConfig.powerPreference;
```

Comment:
`ModelViewerGlobalConfig` is an interface, which means it's just a type, not a real object. It's a decorator to help TS know that `ModelViewerElementBase.powerPreference` exists, but I don't think this logic is doing what you expect.

Reply:
It's true that ModelViewerGlobalConfig is an interface. What it does here is that I say hay in the window/global space I'm going to have a modelviewerelement. Sometimes this object may have initialized this static member. If that's the case, read it. This is literally the pattern used for the other two static properties. Otherwise, how do you explain how this works in loading.ts for draco.

Reply:
Oh, good call.  I just read over the similar code in loading.ts, which apparently I haven't looked at in a long time. Thanks for the explanation!